### PR TITLE
few files to remove

### PR DESCRIPTION
**Pull request by AlexRed**
Pull Request for Issue # .

### Summary of Changes
upgrade from 3.6.5 with current staging these 10 files shall not be present anymore.

some file from #13519

### CODE REVIEW

File: administrator/components/com_admin/script.php [hunk 1]
```diff
@@ -1747,11 +1747,20 @@ public function deleteUnexistingFiles()
 			'/administrator/components/com_modules/layouts/joomla/searchtools/default.php',
 			'/administrator/components/com_templates/layouts/joomla/searchtools/default/bar.php',
 			'/administrator/components/com_templates/layouts/joomla/searchtools/default.php',
-			// Joomla 3.7.0
```
Comment (infograf768) [inline administrator/components/com_admin/script.php]: Any reason to delete this one?

Comment (infograf768) [inline administrator/components/com_admin/script.php]: No p. we already have that comment line 1698

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 2e27c66da87526c3f3213a352d9d8afb67ebb371<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14985">issues.joomla.org/tracker/joomla-cms/14985</a>.</sub>

File: administrator/components/com_admin/script.php (hunk 1 above)
Comment (AlexRed) [inline administrator/components/com_admin/script.php]: yes

**Comment by infograf768** [comment]
@rdeutz 
this should be merged on review

**Comment by infograf768** [comment]
@wilsonge we do not get other testers here but this is important.

**Comment by infograf768** [comment]
@AlexRed 

This patch has conflicts. Can you correct?

**Pull request merged**